### DISCUSSION TITLE
Extract only a specific file

### DISCUSSION THREAD

**Discussion by demhademha** [nmoinvaz/minizip#570] [Q&A]
Is it possible to make minizip only extract a specific file in a folder. 
Imagine I have the zip called `example.zip`, which has several files in it. Imagine that I only want to extract the file `a.mp4`, how would I go around doing this?
Regards

**Comment by nmoinvaz** [comment]
The `minizip_extract` functon in minizip.c shows how to use a wildcard pattern to extract specific files. If you only wanted to match one file then you would use the filename instead of a wildcard pattern.
https://github.com/zlib-ng/minizip-ng/blob/bc93e4077b542a35e9c60f275d984fc7690163dd/minizip.c#L351
